src/main/java/uk/gov/register/resources/PreviewItemsResource.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.register.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import io.dropwizard.views.View;
+import uk.gov.register.core.HashingAlgorithm;
+import uk.gov.register.core.Item;
+import uk.gov.register.core.RegisterReadOnly;
+import uk.gov.register.util.HashValue;
+import uk.gov.register.views.ViewFactory;
+import uk.gov.register.views.representations.ExtraMediaType;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Path("/")
+public class PreviewItemsResource {
+
+    protected final ViewFactory viewFactory;
+    private final RegisterReadOnly register;
+
+    @Inject
+    public PreviewItemsResource(final ViewFactory viewFactory, final RegisterReadOnly register) {
+        this.viewFactory = viewFactory;
+        this.register = register;
+    }
+
+    @GET
+    @Path("/preview-items-json/sha-256:{item-hash}")
+    @Produces(ExtraMediaType.TEXT_HTML)
+    @Timed
+    public View jsonByKeyPreview(@PathParam("item-hash") final String itemHash) {
```

Comment:
As in `PreviewEntriesResource`, I'd combine these three separate endpoints into one, taking the `media-type` as a parameter.